### DISCUSSION TITLE
Corkey works for 121?

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#950] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So I did corkey corrupts extension and it said the extension was corrupted but when I restart my chrombook it didn't work anymore.

**Comment by AshtonDavies** [comment, marked as answer]
Do it again after every restart.

**Reply by v4pl** [reply]
WDYM?

**Reply by v4pl** [reply]
?

**Reply by funkymokey** [reply]
repeat the exploit every time you restart your computer

**Comment by Endy3838** [comment]
> So I did corkey corrupts extension and it said the extension was corrupted but when I restart my chrombook it didn't work anymore.

> ?

you have to repeat each time you restart your computer. otherwise the extensions will be installed again which will make you restart  the whole power wash thing again.

This is the only exploit I use to get on  blocked websites
